AK/MemoryStream.cpp

```diff
@@ -54,6 +54,17 @@ ErrorOr<Bytes> FixedMemoryStream::read_some(Bytes bytes)
     return bytes.trim(to_read);
 }
 
+ErrorOr<void> FixedMemoryStream::read_until_filled(AK::Bytes bytes)
+{
+    if (remaining() < bytes.size())
+        return Error::from_string_view_or_print_error_and_return_errno("Can't read past the end of the stream memory"sv, EINVAL);
```

Comment:
This is not an error exposed from the Kernel, so you should just use `from_string_view` or `EINVAL`.

Reply:
> This is not an error exposed from the Kernel

Are you sure? There is no reason why it couldn't be.